core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -551,18 +566,34 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
   }
 
   @Test
-  def testCreatePermissionNeededForWritingToNonExistentTopic() {
-    val newTopic = "newTopic"
+  def testCreatePermissionOnTopicToWriteToNonExistentTopic() {
+    testCreatePermissionNeededToWriteToNonExistentTopic("newTopic",
+      Set(new Acl(userPrincipal, Allow, Acl.WildCardHost, Write), new Acl(userPrincipal, Allow, Acl.WildCardHost, Create)),
```

Comment:
Why do we need `Write` here?

Reply:
thanks, going to simplify by removing the constant arguments